### DISCUSSION TITLE
Ability for screenshots of live sessions

### DISCUSSION THREAD

**Discussion by TazorDE** [f1multiviewer/issue-tracker#62] [Feature requests] [answered]
I often find myself taking OCRs or Screenshots of Race Control messages to use in conversations with friends or as some sort of quick documentation for myself. 
Back when f1viewer with mpv still worked this was a non issue, but currently the only option to do this is to either run Chrome without hardware acceleration or use Firefox. I am currently using the latter option, but I am very intrigued to use the features f1multiviewer brings to the table. But due to it relying on electron there is not a really viable way for me to disable hardware acceleration. 
If you guys could find a simple way to implement a screenshot feature (maybe using chromiums inbuilt version that seems to do work when activating it with the devtools) I would really appreciate it.

**Comment by f1multiviewer** [comment, marked as answer]
This is because of DRM. Even if it was possible, I will not add anything that circumvents the protections put in place by F1TV to avoid having the app taken down because if it, sorry!
